src/utility/pseudo_random.cpp

```diff
@@ -62,30 +76,27 @@ static std::mt19937& get_twister()
     return *twister;
```

Comment:
Above comment should be:
`// This is thread safe because the instance is thread static.`
must have got lost in rebase.